src/index.js

```diff
@@ -59,11 +60,15 @@ history.listen((location) => {
   trackNavigation(location);
 });
 
+// Choose hash router for cdn only
+const Router = process.env.REACT_APP_CDN_BUILD === 'true' ? CdnRouter : ServerRouter;
```

Comment:
What do you think about renaming this variable to the specific behavior, something like `REACT_APP_HASH_ROUTER`, the same for the router names: `BrowserRouter` and `HashRouter`.

Reply:
Or another option could be using a name in place of `'true'`, for example:

```suggestion
const Router = process.env.REACT_APP_ROUTER === 'hash' ? HashRouter : BrowserRouter;
```

Reply:
Yes I think you are about right, second option fits best.